src/openrct2/ride/gentle/MiniGolf.cpp

```diff
@@ -822,8 +832,16 @@ static void paint_mini_golf_hole_c(paint_session * session, uint8 rideIndex, uin
         paint_util_push_tunnel_right(session, height, TUNNEL_10);
     }
 
-    rct_xy16 boundBox       = (direction & 1) ? (rct_xy16){ 26, 32 } : (rct_xy16){ 32, 26 };
-    rct_xy16 boundBoxOffset = (direction & 1) ? (rct_xy16){ 3, 0 } : (rct_xy16){ 0, 3 };
+    if (direction & 1)
+    {
+        rct_xy16 boundBox       = { 26, 32 };
```

Comment:
you re-declare the variable here

Reply:
Which Xcode, for some reason, decided to allow :'(